gradle.properties

```diff
@@ -1 +1,5 @@
-release.scope=patch
\ No newline at end of file
+release.scope=patch
+org.gradle.caching = true
+org.gradle.parallel = true
+org.gradle.configureondemand = true
```

Comment:
This is incubating, so I would not enable just yet, can you please comment it out and explain we nay turn it out once it matures?

Reply:
The configuration on demand feature is incubating. If projects are configured as what it asked, it would be a good configuration for project build.
I fully understand your concern about stability, so we can just try to remove it

Reply:
Thanks.